### PR TITLE
🐛 Source GItHub: Increase backoff for stream `Contributor Activity`

### PR DESCRIPTION
**Pull request by artem1205**
## What

Resolve https://github.com/airbytehq/oncall/issues/3120

## How

- increase backoff 
- update docs

## Recommended reading order
1. `streams.py`
2. `github.md`

## 🚨 User Impact 🚨

no breaking changes

## Pre-merge Actions

<details><summary><strong>Updating a connector</strong></summary>

### Community member or Airbyter

- Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- Unit & integration tests added

### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items.

- Create a non-forked branch based on this PR and test the below items on it
- Build is successful
- If new credentials are required for use in CI, add them to GSM. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci).

</details>

### CODE REVIEW

File: airbyte-integrations/connectors/source-github/Dockerfile [hunk 1]
```diff
@@ -12,5 +12,5 @@ RUN pip install .
 ENV AIRBYTE_ENTRYPOINT "python /airbyte/integration_code/main.py"
 ENTRYPOINT ["python", "/airbyte/integration_code/main.py"]
 
-LABEL io.airbyte.version=1.4.4
+§LABEL io.airbyte.version=1.4.5
```
Comment (davydov-d) [inline airbyte-integrations/connectors/source-github/Dockerfile:15]: Typo

**Review by davydov-d — APPROVED** [review]

**Comment by airbyte-oss-build-runner** [comment]
##  source-github test report (commit [`396df12ed2`](https://github.com/airbytehq/airbyte/pull/31023/commits/396df12ed2880245f8a4faf88a5e211eb570ceb4)) - ✅

⏲️  Total pipeline duration: 08mn04s 

| Step                                                       | Result   |
|:-----------------------------------------------------------|:---------|
| Connector package install                                  | ✅        |
| Build source-github docker image for platform linux/x86_64 | ✅        |
| Unit tests                                                 | ✅        |
| Acceptance tests                                           | ✅        |
| Code format checks                                         | ✅        |
| Validate metadata for source-github                        | ✅        |
| Connector version semver check                             | ✅        |
| Connector version increment check                          | ✅        |
| QA checks                                                  | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/artem1205_source-github-add-docs/1696355270/396df12ed2880245f8a4faf88a5e211eb570ceb4/source-github/1.4.5/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:396df12ed2880245f8a4faf88a5e211eb570ceb4)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-github test
```

**Comment by airbyte-oss-build-runner** [comment]
##  source-github test report (commit [`5a0836e984`](https://github.com/airbytehq/airbyte/pull/31023/commits/5a0836e9841c0317410345fec9ef178e72ffb910)) - ❌

⏲️  Total pipeline duration: 07mn14s 

| Step                                                       | Result   |
|:-----------------------------------------------------------|:---------|
| Connector package install                                  | ✅        |
| Build source-github docker image for platform linux/x86_64 | ✅        |
| Unit tests                                                 | ✅        |
| Acceptance tests                                           | ❌        |
| Code format checks                                         | ✅        |
| Validate metadata for source-github                        | ✅        |
| Connector version semver check                             | ✅        |
| Connector version increment check                          | ✅        |
| QA checks                                                  | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/artem1205_source-github-add-docs/1696362138/5a0836e9841c0317410345fec9ef178e72ffb910/source-github/1.4.5/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:5a0836e9841c0317410345fec9ef178e72ffb910)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-github test
```

**Comment by airbyte-oss-build-runner** [comment]
##  source-github test report (commit [`254f1d6951`](https://github.com/airbytehq/airbyte/pull/31023/commits/254f1d69518dfe45383dc1f2d10283cfc6043de0)) - ❌

⏲️  Total pipeline duration: 02mn59s 

| Step                                                       | Result   |
|:-----------------------------------------------------------|:---------|
| Connector package install                                  | ✅        |
| Build source-github docker image for platform linux/x86_64 | ✅        |
| Unit tests                                                 | ✅        |
| Acceptance tests                                           | ❌        |
| Code format checks                                         | ✅        |
| Validate metadata for source-github                        | ✅        |
| Connector version semver check                             | ✅        |
| Connector version increment check                          | ✅        |
| QA checks                                                  | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/artem1205_source-github-add-docs/1696363461/254f1d69518dfe45383dc1f2d10283cfc6043de0/source-github/1.4.5/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:254f1d69518dfe45383dc1f2d10283cfc6043de0)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-github test
```

**Review by darynaishchenko — APPROVED** [review]

**Comment by airbyte-oss-build-runner** [comment]
##  source-github test report (commit [`20594abc69`](https://github.com/airbytehq/airbyte/pull/31023/commits/20594abc69937315ad98f5ec8f9e87e08b158c46)) - ✅

⏲️  Total pipeline duration: 08mn05s 

| Step                                                       | Result   |
|:-----------------------------------------------------------|:---------|
| Connector package install                                  | ✅        |
| Build source-github docker image for platform linux/x86_64 | ✅        |
| Unit tests                                                 | ✅        |
| Acceptance tests                                           | ✅        |
| Code format checks                                         | ✅        |
| Validate metadata for source-github                        | ✅        |
| Connector version semver check                             | ✅        |
| Connector version increment check                          | ✅        |
| QA checks                                                  | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/artem1205_source-github-add-docs/1696409085/20594abc69937315ad98f5ec8f9e87e08b158c46/source-github/1.4.5/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:20594abc69937315ad98f5ec8f9e87e08b158c46)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-github test
```

**Comment by airbyte-oss-build-runner** [comment]
##  source-github test report (commit [`4eda1132fd`](https://github.com/airbytehq/airbyte/pull/31023/commits/4eda1132fdf7259657346293b5b4817fae231505)) - ✅

⏲️  Total pipeline duration: 07mn16s 

| Step                                                          | Result   |
|:--------------------------------------------------------------|:---------|
| Connector package install                                     | ✅        |
| Build source-github docker image for platform(s) linux/x86_64 | ✅        |
| Unit tests                                                    | ✅        |
| Acceptance tests                                              | ✅        |
| Code format checks                                            | ✅        |
| Validate metadata for source-github                           | ✅        |
| Connector version semver check                                | ✅        |
| Connector version increment check                             | ✅        |
| QA checks                                                     | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/artem1205_source-github-add-docs/1696437809/4eda1132fdf7259657346293b5b4817fae231505/source-github/1.4.5/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:4eda1132fdf7259657346293b5b4817fae231505)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-github test
```

**Pull request merged**